app/views/needs/index.html.erb

```diff
@@ -4,9 +4,20 @@
   <h1 class="page-title">Needs <span class="page-title__count">(<%= @needs.total_count %>)</span></h1>
 </div>
 
+     
+
+<%= form_with(url: needs_url, method: :get, local: true) do %>
```

Comment:
I think you may have moved this outside of `<details>` accidentally. It may well break the styling, and the `end` tag (which renders `</form>`) is still inside of `<details>` so would produce invalid markup.